RecoTauTag/RecoTau/plugins/RecoTauJetRegionProducer.cc

```diff
@@ -149,6 +149,11 @@ void RecoTauJetRegionProducer::produce(edm::Event& evt, const edm::EventSetup& e
 	  pfCand != pfCands.end(); ++pfCand ) {
       bool isMappedToJet = false;
       if ( pfCandAssocMapSrc_.label() != "" ) {
+	auto temp = jetToPFCandMap->find(jetRef);
+	if( temp == jetToPFCandMap->end() ) {
+	  edm::LogWarning("WeirdCandidateMap") << "Candidate map for jet " << jetRef.key() << " is empty!";
```

Comment:
It appears to be a pathology for high pt 4-muon guns when muons are close.
Regular physics samples not affected. Checked 1k events that, didn't see
anything.